src/domain/place/unitSelector/UnitPlaceSelector.tsx

```diff
@@ -74,7 +74,9 @@ const UnitPlaceSelector: React.FC<Props> = ({
         value: place.id || '',
       }))
       // Filter the results with a search value!
-      .filter((place) => place.label.includes(searchValue)) || [];
+      .filter((place) =>
+        place.label.toLowerCase().includes(searchValue.toLowerCase())
+      ) || [];
```

Comment:
I also tried a really lightweight plugin that uses `localeCompare()` with `includes()`. It worked, but the field and the input became really laggy, so I decided to go with really fast and simple `toLowerCase()`.
```suggestion
      # import { localeIncludes } from 'locale-includes';
      .filter((place) =>
        localeIncludes(place.label, searchValue, {
          usage: 'search',
          sensitivity: 'base',
        })
      ) || [];
```